EIPS/eip-4410.md

```diff
@@ -0,0 +1,170 @@
+---
+eip: 4410 
+title:  NFT Memories Standard
+description: A specification for storing memories associated with NFTs as blockchain events.
+author: Ken Woodruff (@kennethhoytwoodruff), David Menard (@davidmd222), Rocky Melchor (@rocky-m)
+discussions-to: https://github.com/RealItems/EIPs/issues/1
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-10-27
+---
+
+## Abstract
+
+This standard allows contracts, such as NFTs that support [ERC-721](./eip-721.md) and [ERC-1155](./eip-1155.md) interfaces, to add 'memories' to an NFT.  This is intended for NFT dApps and marketplaces that want to support memories on top of existing NFT standards. Memories are stored as events by calling a method on the smart contract containing the NFT.  This ERC should be considered a minimal, gas-efficient building block for extending the [ERC-721](.eip-721.md) and [ERC-1155](.eip-1155.md) standards.
```

Comment:
I don't know what a memory is in this context :sweat_smile: 

Might be worth adding a definition to the abstract?